demos/common/export_models/requirements.txt

```diff
@@ -2,14 +2,14 @@
 --extra-index-url "https://storage.openvinotoolkit.org/simple/wheels/nightly"
 --extra-index-url "https://storage.openvinotoolkit.org/simple/wheels/pre-release"
 --pre
-optimum-intel
+optimum-intel@git+https://github.com/huggingface/optimum-intel.git
 openvino-tokenizers<=2025.3.0.0.dev20250812
 openvino<=2025.3.0.dev20250812
 nncf>=2.11.0 
 sentence_transformers
 sentencepiece==0.2.0
 openai 
-transformers<4.53
+transformers<4.54
```

Comment:
shouldn't it also specify transformers==4.54 ? so earlier version wouldn't be installed

Reply:
@ngrozae  we don't want to include 4.54. It should be 4.53.x or older.